src/discrete-layer/models/discreteLayersStore.ts

```diff
@@ -45,14 +50,41 @@ export const discreteLayersStore = types
           // eslint-disable-next-line @typescript-eslint/ban-ts-comment
           // @ts-ignore
           const result = yield  Promise.resolve(MOCK_DATA_IMAGERY_LAYERS_ISRAEL);
-          self.layersImages = result.map(item => ({...item, selected:false}));
+          self.layersImages = filterBySearchParams(result).map(item => ({...item, selected:false}));
```

Comment:
createMockData - not in use fix lint
![image](https://user-images.githubusercontent.com/18442671/103364939-a1e4b600-4ac7-11eb-8a45-1868d8af6d30.png)

Reply:
It's a generic way to mimic data bulks, generates as much as defined entries.
remain code unchanged.

Till we in massive development and not all decisions are taken I prefer to leave code with my comments and remarks.